src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/FlatFWTest.java

```diff
@@ -18,39 +18,49 @@
 import static java.nio.ByteBuffer.allocateDirect;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertSame;
 
 import org.agrona.MutableDirectBuffer;
 import org.agrona.concurrent.UnsafeBuffer;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 import org.reaktivity.reaktor.internal.test.types.inner.FlatFW;
-import org.reaktivity.reaktor.internal.test.types.StringFW;
+import org.reaktivity.reaktor.internal.test.types.inner.FlatFW.Builder;
 
 public class FlatFWTest
 {
-    private final MutableDirectBuffer buffer = new UnsafeBuffer(allocateDirect(100))
+    private static final int INDEX_FIXED1 = 0;
+
+    private static final int INDEX_FIXED2 = 1;
+
+    private static final int INDEX_STRING1 = 2;
+
+    private static final int INDEX_FIXED3 = 3;
+
+    private static final int INDEX_STRING2 = 4;
+
+    private final MutableDirectBuffer buffer = new UnsafeBuffer(allocateDirect(100));
     {
-        {
-            // Make sure the code is not secretly relying upon memory being initialized to 0
-            setMemory(0, capacity(), (byte) 0xab);
-        }
-    };
+    // Make sure the code is not secretly relying upon memory being initialized to 0
+    buffer.setMemory(0, buffer.capacity(), (byte) 0xab);
+    }
+
+    MutableDirectBuffer expected = new UnsafeBuffer(allocateDirect(100));
+    {
+        // Make sure the code is not secretly relying upon memory being initialized to 0
+        expected.setMemory(0, expected.capacity(), (byte) 0xab);
+    }
     private final FlatFW.Builder flatRW = new FlatFW.Builder();
     private final FlatFW flatRO = new FlatFW();
-    private final StringFW.Builder stringRW = new StringFW.Builder();
-    private final MutableDirectBuffer valueBuffer = new UnsafeBuffer(allocateDirect(100));
 
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
 
     @Test
     public void shouldProvideTypeId() throws Exception
     {
-        int limit = flatRW.wrap(buffer, 0, 100)
-                .fixed1(10)
-                .string1("value1")
-                .string2("value2")
+        int limit = setRequiredFields(flatRW.wrap(buffer, 0, 100), Integer.MAX_VALUE)
```

Comment:
Use buffer.capacity() instead of hard coding the value 100, because later when we generate tests the value 100 will not always be sufficient. Please apply this comment in any other similar places.